### PR TITLE
Bump build number of GraphQL version

### PR DESCRIPTION
**Pull request by Shane32**
- Gets the latest bug fixes for subscription handling

### CODE REVIEW

**Comment by Shane32** [comment]
Will release this as 7.4.0 since there is an API change -- ok @sungam3r ?

**Comment by sungam3r** [comment]
Let's publish GraphQL.NET v7.4 and reference it here.

File: Directory.Build.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>7.3.0-preview</VersionPrefix>
+    <VersionPrefix>7.5.0-preview</VersionPrefix>
```
Comment (sungam3r) [inline Directory.Build.props:4]: 7.5?

Comment (Shane32) [reply Directory.Build.props:4]: this should always be at least one version ahead.  if we publish 7.4 then this needs to be at least 7.5.  as far as i'm concerned we could just set it to 7.99.0-preview and leave it be.  The only reason it needs to be ahead of the nuget version at all is when referencing a dependency that depends on this package, such as when referencing GraphQL Server 7.3.0 but trying to use a GraphQL.NET preview.  If the GraphQL.NET preview number was 7.2.0-preview then nuget would attempt to bump the nuget package of GraphQL.NET when restoring packages.  But if GraphQL.NET was 7.99.0-preview then there would never be a problem.

Comment (sungam3r) [reply Directory.Build.props:4]: I would switch to MinVer to manage version automatically.

Comment (Shane32) [reply Directory.Build.props:4]: Yeah, I'm not a fan of adding more build-time dependencies when they are not necessary.  Our system works perfectly now and it took me longer to write these posts than any number of version bumps over the last couple years.

**Comment by sungam3r** [comment]
@Shane32 OK to release GraphQL.NET v7.4 and reference it here?

**Comment by sungam3r** [comment]
>  Then both will be 7.4

This is the reason why I suggest to bump. I'll publish release.

**Comment by sungam3r** [comment]
https://github.com/graphql-dotnet/graphql-dotnet/releases/tag/7.4.0

Also I need v7.4 in [my own project](https://github.com/sungam3r/graphql-introspection-model) and https://github.com/sungam3r/graphql-sdl-exporter to integrate new deprecation feature.

File: Directory.Build.props [hunk 2]
```diff
@@ -32,7 +32,7 @@
     <RootNamespace>GraphQL.Server.$(MSBuildProjectName)</RootNamespace>
     <PackageId>GraphQL.Server.$(MSBuildProjectName)</PackageId>
 
-    <GraphQLVersion>7.3.0</GraphQLVersion>
+    <GraphQLVersion>7.3.1</GraphQLVersion>
```
Comment (Shane32) [inline Directory.Build.props:35]:
```suggestion
    <GraphQLVersion>7.4.0</GraphQLVersion>
```

**Review by sungam3r — APPROVED** [review]

**Pull request merged**